### DISCUSSION TITLE
Use xdg based directory for termux specific packages

### DISCUSSION THREAD

**Discussion by amstests** [termux/termux-packages#13258] [Ideas]
Is posible to termux follow [Xdg based directory](https://wiki.archlinux.org/title/XDG_Base_Directory), and move ~/.termux to ~/.config/termux?
Its will be nice..

1. make $HOME more cleaner from .dotfiles
2. user can easy split backup (just config files) by type `tar -cvzf com.termux.config.tar.gz $HOME/.config`,
3. it will probably support some packages from x11-packages

Notes:
I mean specific packages for termux, like termux-tools, termux-am, termux-api and others that use the ~/.termux directory

Best regards.

**Comment by ckysmrn** [comment]
I agree to point 2, so +1
